src/latlon.py

```diff
@@ -12,14 +12,18 @@ def getlatlon(imgpath):
     im = rasterio.open(imgpath)
     crs = im.crs.__str__()  # Coordinate reference system code
     nrows, ncols = im.shape
-    rows, cols = np.meshgrid(np.arange(nrows), np.arange(ncols))
+    cols, rows = np.meshgrid(np.arange(ncols), np.arange(nrows))
     xs, ys = rasterio.transform.xy(im.transform, rows, cols)
+    xs = np.array(xs)
+    ys = np.array(ys)
```

Comment:
Is the original data type in `xs`, and `ys` different than `np.array`? Again, just curious.

Reply:
Yes, it's a list

Reply:
Regarding the test -- no, I did not write a test function. The way the Julia test stuff works is still pretty opaque to me. I just set up the test data so that it would be more straightforward to write the test function later.